hypha/apply/users/models.py

```diff
@@ -229,13 +229,18 @@ def get_full_name(self):
     def get_short_name(self):
         return self.email
 
-    def get_full_name_with_group(self):
+    def get_display_name_with_group(self) -> str:
+        """Gets the user's display name, along with their role in parenthesis
+
+        If the user has a full name set that will be used, otherwise pulls the email.
+        """
+        display_name = self.full_name if self.full_name else self.email
         is_apply_staff = f" ({STAFF_GROUP_NAME})" if self.is_apply_staff else ""
         is_reviewer = f" ({REVIEWER_GROUP_NAME})" if self.is_reviewer else ""
         is_applicant = f" ({APPLICANT_GROUP_NAME})" if self.is_applicant else ""
         is_finance = f" ({FINANCE_GROUP_NAME})" if self.is_finance else ""
         is_contracting = f" ({CONTRACTING_GROUP_NAME})" if self.is_contracting else ""
-        return f"{self.full_name.strip()}{is_apply_staff}{is_reviewer}{is_applicant}{is_finance}{is_contracting}"
+        return f"{display_name.strip()}{is_apply_staff}{is_reviewer}{is_applicant}{is_finance}{is_contracting}"
```

Comment:
I think we can remove "strip()" from this line. It is already done in "get_full_name".